### PR TITLE
powershell common

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/20102
fixes: https://github.com/demisto/etc/issues/20103

## Description
Move $demisto into common file. Add some wrapper utility functions. First powershell automation (VerifyJSON) with unit testing. Requires updated demisto-sdk.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [x] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [x] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by glicht** [comment]
@kirbles19 please review:
* yml file: https://github.com/demisto/content/edit/powershell-common/Scripts/VerifyJSON/VerifyJSON.yml
* changelog: https://github.com/demisto/content/edit/powershell-common/Scripts/CommonServerPowerShell/CHANGELOG.md

**Comment by yardensade** [comment]
Look good,
My main concern here is what happens in server after the merge and from that point forward, if something breaks in common it will break server build due to all the unit tests for demisto object.

File: Packs/Base/.secrets-ignore [hunk 1]
```diff
@@ -14,4 +14,4 @@ https://exchange.xforce.ibmcloud.com
 user@example.com
 8.8.8.8
 https://google.com
-https://example.com
\ No newline at end of file
+https://example.comhttps://docs.microsoft.com
```
Comment (yardensade) [inline Packs/Base/.secrets-ignore]:
```suggestion
https://example.com
https://docs.microsoft.com
```

**Comment by kirbles19** [comment]
@glicht Done.

**Pull request merged**